docs/content/connecting-to-spatialos.md

```diff
@@ -5,14 +5,14 @@
 
 _This document relates to both [MonoBehaviour and ECS workflows]({{urlRoot}}/content/intro-workflows-spatialos-entities)._
 
-The SpatialOS [Runtime]({{urlRoot}}/content/glossary#spatialos-runtime) manages your game world by keeping track of all [SpatialOS entities](({{urlRoot}}/content/glossary#spatialos-entity) and the current state of their [components]({{urlRoot}}/content/glossary#spatialos-component).
+The SpatialOS [Runtime]({{urlRoot}}/content/glossary#spatialos-runtime) manages your game world by keeping track of all [SpatialOS entities]({{urlRoot}}/content/glossary#spatialos-entity) and the current state of their [components]({{urlRoot}}/content/glossary#spatialos-component).
 To execute any kind of logic on these entities, we use [workers]({{urlRoot}}/content/glossary#worker). You can set up many different types of workers but the two main types are  [server-workers]({{urlRoot}}/content/glossary#server-worker) and [client-workers]({{urlRoot}}/content/glossary#client-worker).
 
 (To find out more about workers in the GDK, see the introduction to [workers in the GDK]({{urlRoot}}/content/workers/workers-in-the-gdk).)
 
 It’s the workers (both server-workers and client-workers) which create a connection between your game and the SpatialOS Runtime. In order for them to do this, you need to set up the configuration of your worker types as part of your game development. Then, when your game runs, it creates worker instances which connect to the SpatialOS Runtime.  During their creation, worker instances attempt to connect to the SpatialOS Runtime. If the connection fails, the creation of the worker instance fails.
 
-If you are using the [MonoBehaviour workflow or the ECS workflow]{{urlRoot}}/content/intro-workflows-spatialos-entities), you can use the [Worker Connector]({{urlRoot}}/content/gameobject/linking-spatialos-entities) or the [Worker API]({{urlRoot}}/content/workers/api-worker) to set up your workers (server-workers and client-workers) to connect to SpatialOS. Upon successfully connecting to the SpatialOS Runtime, your worker stores a `Connection` object.
+If you are using the [MonoBehaviour workflow or the ECS workflow]{{urlRoot}}/content/intro-workflows-spatialos-entities), you can use the [Worker Connector]({{urlRoot}}/content/gameobject/linking-spatialos-entities) or the `Worker` to set up your workers (server-workers and client-workers) to connect to SpatialOS. Upon successfully connecting to the SpatialOS Runtime, your worker stores a `Connection` object.
```

Comment:
I think this might be fine for now, but ideally we don't want users to have to know there is a `Connection` object as it should be completely hidden.